test/Microsoft.TestTemplates.AcceptanceTests/DotnetCoreTemplateTests.cs

```diff
@@ -29,6 +29,9 @@ public class DotnetCoreTemplateTests : AcceptanceTestBase
             "MSTest-CSharp",
```

Comment:
Unrelated to this change, but should we include 5.0 in the list of .NET Core Versions we want to validate as well?

Reply:
Yes, last time I was doing changes in that area, net5.0 was very early preview, and I was releasing this for the first time, so I postponed it.

Reply:
https://github.com/dotnet/test-templates/issues/95